### PR TITLE
fix(trajectory_follower_node): fix config file of plot juggler

### PR DESCRIPTION
**Pull request by tkimura4**
## Description
I adjusted the formatting to enable the loading of the plot juggler's config file, which was previously failing to load.

## Tests performed

image

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by TakaHoribe — APPROVED** [review]

**Review by Owen-Liuyuxuan — APPROVED** [review]
I have tested on my local setup with PlotJuggler-3 `v3.9.1`

File: control/trajectory_follower_node/config/plot_juggler_trajectory_follower.xml [hunk 1]
```diff
@@ -241,33 +241,42 @@
...
+   <scripts/>
+  </plugin>
+  <plugin ID="CSV Exporter"/>
+  <plugin ID="ROS2 Topic Re-Publisher"/>
  </Plugins>
  <!-- - - - - - - - - - - - - - - -->
  <previouslyLoaded_Datafiles/>
- <previouslyLoaded_Streamer name="ROS 2 Topic Subscriber"/>
+ <previouslyLoaded_Streamer name="ROS2 Topic Subscriber"/>
```
Comment (tkimura4) [inline control/trajectory_follower_node/config/plot_juggler_trajectory_follower.xml:279]: It seems that we can no longer use "ROS 2 Topic Subscriber" as before.
image

Comment (tkimura4) [reply control/trajectory_follower_node/config/plot_juggler_trajectory_follower.xml:279]: https://github.com/tier4/autoware-spell-check-dict/pull/720
I have created a pull request to autoware-spell-check-dict to allow the string 'ROS2'.

Comment (tkimura4) [reply control/trajectory_follower_node/config/plot_juggler_trajectory_follower.xml]: [set spell-check ignore labels of for "ROS2"](https://github.com/autowarefoundation/autoware.universe/pull/6910/commits/456a83c5d507882234002b64c4f2e1d255f7d2d5)

**Pull request merged**